### PR TITLE
feat(EMQX Streaming): Add docs for EMQX Streaming

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

File: en_US/emqx_streaming/emqx_streaming.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+#### Kafka Consumer Group
+
+A **Kafka Consumer Group** is a mechanism that allows multiple Kafka consumers to cooperate in consuming data from a stream, ensuring that the load is distributed evenly among the group and that each message is processed only once.
+
+### Design Architecture
+
+The architecture diagram illustrates the EMQX Platform and how it handles both MQTT and Kafka message streams.
+
+![architecture](./_assets/architecture.png)
```
Comment (daleiz) [inline en_US/emqx_streaming/emqx_streaming.md:87]: remove this figure, it is not suitable here.

File: en_US/price/plans.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The Premium Plan offers all the advantages of the Dedicated Plan, along with add
 
 **Cluster Linking**: Facilitates cross-regional communication through cluster bridging.
 
-**Kafka Streaming (coming soon)**: Direct support for Kafka protocol, allowing Kafka clients to consume MQTT messages seamlessly.
+**EMQX Streaming (coming soon)**: Direct support for Kafka protocol, allowing Kafka clients to consume MQTT messages seamlessly.
```
Comment (0721Betty) [inline en_US/price/plans.md:32]: remove coming soon

File: en_US/emqx_streaming/emqx_streaming.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+### Why EMQX Streaming?
+
+While MQTT excels in real-time message distribution and handling many concurrent connections, it lacks built-in support for massive message storage and on-demand access to historical data. Traditionally, users have integrated MQTT brokers with Kafka to meet these needs, but this approach is expensive, complex, and operationally demanding.
+
+The EMQX Streaming feature addresses these challenges by providing a unified solution within the EMQX Platform.
+
+### What is EMQX Streaming?
+
+Building on EMQX's existing [durable storage](https://docs.emqx.com/en/emqx/latest/durability/durability_introduction.html) capabilities, the EMQX Streaming feature makes these stored data streams accessible through interfaces commonly used for data streaming, such as Kafka. It allows users to retrieve and replay stored messages through familiar Kafka clients, the operational overhead of managing a separate Kafka cluster.
```
Comment (daleiz) [inline en_US/emqx_streaming/emqx_streaming.md:25]: need to refine this paragraph.

File: en_US/emqx_streaming/use_emqx_streaming.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+   - **Stream Type**: Select the stream type:
+     - **Default**: The default type Stream is associated with an MQTT topic filter, and MQTT messages matching the topic filter are saved to the Stream. 
+     - **Free**: The free type Stream is not associated with an MQTT topic filter.
+   - **MQTT Topic Filter**: Enter a topic filter for MQTT messages. MQTT messages matching this topic filter are saved to the corresponding default type Stream. For this example, enter `t1/+`.
+
+3. Click **Confirm**.
+
+Once the stream is created, it will appear in the **Streams** list. By clicking the stream name, you can view details, including the partition offsets and other stream-specific information.
+
+<!-- How to configure the free stream partition?-->
```
Comment (0721Betty) [inline en_US/emqx_streaming/use_emqx_streaming.md:43]: image

**Review by daleiz — CHANGES_REQUESTED** [review]

File: en_US/price/plans.md (hunk 2 above)
Comment (daleiz) [reply en_US/price/plans.md:32]: also, change the feature statement like below:
store MQTT messages as data streams and consume them directly using Kafka clients

**Comment by Meggielqk** [comment]
@YangKian/ @0721Betty , would you please help to provide a screenshot in English for the consumer group details view with some simulated data?

**Comment by YangKian** [comment]
> @YangKian/ @0721Betty , would you please help to provide a screenshot in English for the consumer group details view with some simulated data?

OK, I will update all console screenshots to keep them consistent.

File: en_US/emqx_streaming/use_emqx_streaming.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+ The retention period for both types of streams is fixed at 1 day and cannot be modified.
+
+1. Navigate to **Streaming (beta)** -> **Streams**.
+2. Click **New** on the Streams page. On the **New Streams** popup, complete the following settings:
+   - **Stream Name**: Provide a name for the stream, for example, `demo1`. The Kafka topic for a Kafka client must match the stream name.
+   - **Stream Type**: Select the stream type.
+     - **Default**: The default type Stream is associated with an MQTT topic filter, and MQTT messages matching the topic filter are saved to the Stream. 
+     - **Free**: The free type Stream is not associated with an MQTT topic filter and is typically used for other data processing purposes.
+   - **MQTT Topic Filter**: If you select `Default` as the stream type, enter a topic filter for MQTT messages. MQTT messages matching this topic filter are saved to the corresponding default type Stream. For this example, enter `t1/+`.
+   - **Number of Partitions**: If you select `Free` as the stream type, specify the number of partitions to divide the stream for scalability and parallel processing.
```
Comment (YangKian) [inline en_US/emqx_streaming/use_emqx_streaming.md:38]: I think the description of the number of partitions is not very accurate. Perhaps we can remove the content related to `MQTT Topic Filter` and `Number of Partitions`? When users try to create a stream, they will naturally know what content needs to be filled in according to the console's panel prompts.

In addition, consider creating a link pointing to `emqx_streaming.md` to help users who are not familiar with the background gain some necessary background knowledge?  We can write something like: `For detailed information about Kafka Stream, please refer to: <the link>` in the line 30

What do you think? @daleiz @0721Betty

**Comment by YangKian** [comment]
It seems that this picture is no longer needed. `./_assets/architecture.png`

File: en_US/emqx_streaming/use_emqx_streaming.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+This command will consume messages from the specified stream starting from the beginning. You should see the previously published messages being consumed.
+
+![consume_messages](./_assets/consume_messages.png)
+
+## View Consumed Messages in EMQX Platform Console
+
+To verify the consumed messages, navigate to **Streaming (beta)** -> **Consumer Groups** in the EMQX Platform Console.
+
+![consumer_groups](./_assets/consumer_groups.png)
```
Comment (YangKian) [inline en_US/emqx_streaming/use_emqx_streaming.md:85]: image

image

**Comment by YangKian** [comment]
@Meggielqk image updated. If you need any other screenshots, here is the console panel address: https://cloud-intl.dev.en.consoleassets.mqttce.com/console/deployments/hbe1751b/overview,

**Comment by Meggielqk** [comment]
> @Meggielqk image updated. If you need any other screenshots, here is the console address: https://cloud-intl.dev.en.consoleassets.mqttce.com/console/deployments/hbe1751b/overview,

Thank you!

**Pull request merged**